packages/manager/apps/cloud/client/app/dbaas/logs/detail/inputs/add/edit/logs-inputs-add-edit.html

```diff
@@ -75,7 +75,7 @@
             <oui-numeric
                 id="input_port"
                 name="input_port"
-                model="ctrl.input.data.info.exposedPort"
+                data-model="ctrl.input.data.info.exposedPort"
```

Comment:
FYI, you don't have to set `data-` prefixes before custom properties of component

Reply:
Answered in previous comment.